lib/src/services/idb.dart

```diff
@@ -116,6 +116,9 @@ class Idb extends DatabaseService {
 	/// The name for the schedules object store.
 	static const String scheduleStoreName = "schedules";
 
+	/// The name for the default schedules object store.
+	static const String defaultScheduleStoreName = "defaults";
```

Comment:
Ah, okay here's something -- should this really be a separate data store? Think of it like a "table" in SQL or a "collection" in Firestore: can you really say this is a whole collection of similarly-formatted entries? I think this should be filed under "miscellaneous" data somehow -- maybe use the `SharedPreferences` service? Sure, it's a bit weird in that you're using one service from another but I think RamLife can handle it.